lib/lsh--0.0.1.sql

```diff
@@ -1,7 +1,31 @@
 -- complain if script is sourced in psql, rather than via CREATE EXTENSION
-\echo Use "CREATE EXTENSION lsh_index" to load this file. \quit
+\echo Use "CREATE EXTENSION lsh;" to load this file. \quit
 
-CREATE FUNCTION lsh()
-    RETURNS text
-    AS '$libdir/lsh'
-    LANGUAGE C IMMUTABLE STRICT;
\ No newline at end of file
+CREATE OR REPLACE FUNCTION dot(anyarray, anyarray)
```

Comment:
maybe we should use 'lsh' prefix to identify that function is related to this extension?